guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -230,34 +348,62 @@ Depending on the parameter complexity, we distinguish between simple parameters
 [[Parametrized_class]]
 Parametrized class (smart class parameter):: A parameter created by importing a class from Puppet server.
 
+[[Patch-and-release-management]]
+Patch and release management::
+Patch and release management describes the process of acquiring, managing, and installing patches and software updates to your infrastructure.
+Using {Project}, you can keep control of the package versions available to your hosts and deliver applicable errata.
+
 [[Permission]]
 Permission:: Defines an action related to a selected part of {Project} infrastructure (resource type).
 Each resource type is associated with a set of permissions, for example the *Architecture* resource type has the following permissions: *view_architectures*, *create_architectures*, *edit_architectures*, and *destroy_architectures*.
 You can group permissions into roles and associate them with users or user groups.
 
 [[Product]]
-Product:: A collection of content repositories.
-Products are either provided by Red{nbsp}Hat CDN or created by the {Project} administrator to group custom repositories.
+Product::
+Products are named collections of repositories.
+ifdef::satellite[]
+Products are either provided by Red{nbsp}Hat CDN or created to group custom repositories.
+endif::[]
+One or more products form a content view, for example to bundle `{client-os}` and `{client-os} {client-os-major} Client`.
```

Comment:
Content views bundle individual repositories, not products. I'd remove this.

Reply:
Thanks, dropped.